delfin/api/v1/storages.py

```diff
@@ -192,6 +192,9 @@ def _storage_exist(self, context, access_info):
 
         return False
 
+    def perf_collect(self, storage_id, interval, is_historic, resource):
```

Comment:
Add some description for this fucntion ? Do we need this function here in api/v1/storages?